cmd/mvpage/mover.go

```diff
@@ -58,7 +58,12 @@ func (m *mover) move(srcFile string, dstFile string) error {
 		return err
 	}
 
-	if m.srcLinkRegex, err = regexp.Compile(regexp.QuoteMeta("(" + m.srcLink)); err != nil {
+	if _, err = os.Stat(m.dstFile); err == nil {
+		return fmt.Errorf("destination '%s' already exists", m.dstFile)
+	}
+
+	re := `\((` + regexp.QuoteMeta(m.srcLink) + `(/|))(#.*|)\)`
```

Comment:
very nice

Reply:
A PITA to write though :-). Thank goodness for https://regex101.com/ :-)